### DISCUSSION TITLE
Can I use vitest in project that doesn't use vite?

### DISCUSSION THREAD

**Discussion by I-am-tk** [vitest-dev/vitest#3362] [Q&A]
I am trying to use vitest in my node project. How can configure it to run without vite?

**Comment by ghiscoding** [comment]
it doesn't require Vite, you can certainly use Vitest as a standalone, the only difference is to use `vitest.config.ts` instead of `vite.config.ts` (describe in [docs](https://vitest.dev/guide/#configuring-vitest)). I use Vitest in [Lerna-Lite](https://github.com/lerna-lite/lerna-lite) which is a NodeJS project without any web stuff, so yes you can use Vitest in Node project without Vite.

**Reply by IanBellomy** [reply]
Is this still the case? Top of docs have a dedicated callout saying “Vitest requires Vite >=v6.0.0 and Node >=v20.0.0”

**Reply by AriPerkkio** [reply]
Yes. There's also [`--experimental.viteModuleRunner=false`](https://vitest.dev/config/experimental.html#experimental-vitemodulerunner) if you don't want to use any Vite features. At the moment Vite is still installed as peer dependency.

**Reply by IanBellomy** [reply]
Roger, thanks!
